cfn-templates/cid-cfn.yml

```diff
@@ -515,6 +528,15 @@ Resources:
                   s3path = s3path.replace('{account_id}', account_id)
                   parts = s3path.split('/')
                   data['Bucket'] = parts[0]
+                  if not bucket_exists(data['Bucket']): raise Exception('Bucket in CURBucketPath does not exist. Please deploy CUR, just creating a bucket is not enough')
+                  # detect Type of CUR and choose the right partitions structure
+                  if len(parts[1:]) == 1: # most likely it is created by CFN or similar
```

Comment:
If customer set up CUR less then 24 hours before this will fail. We should not make them wait 24 hours between CUR set up and dashboards set up. Many customers allocate time for deployment and they are in focus of deployment and it's better to allow them to deploy this in one go. Could you please check that bucket exists for now and drop part with partitions check? Additionally you can check of existence of aws-programmatic-access-test-object which is delivered immediately at CUR creation, so it will be a solid evidence that it's a cur bucket

Reply:
It will not. It is just checking the existence of the bucket and the path provided as paramter